tests/formats/cml.cpp

```diff
@@ -0,0 +1,78 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#include <streambuf>
+#include <fstream>
+
+#include "catch.hpp"
+#include "helpers.hpp"
+#include "chemfiles.hpp"
+using namespace chemfiles;
+
+#include <boost/filesystem.hpp>
+namespace fs=boost::filesystem;
+
+#include <iostream>
+
+TEST_CASE("Read files in CML format") {
+    SECTION("Check nsteps") {
+        Trajectory file1("data/cml/ethane.cml");
+        CHECK(file1.nsteps() == 1);
+
+        Trajectory file2("data/cml/drugs.cml");
+        CHECK(file2.nsteps() == 4);
+    }
+
+    SECTION("Read next step") {
+        Trajectory file("data/cml/drugs.cml");
+        auto frame = file.read();
+        CHECK(frame.size() == 17);
+        CHECK(frame.get("name")->as_string() == "naproxen");
+
+        // Check positions
+        auto positions = frame.positions();
+        CHECK(approx_eq(positions[0]  == Vector3D(0.0000, -4.6200, 0.0000), 1e-3));
```

Comment:
This is not doing what you think it does =) It should be `approx_eq(positions[0], Vector3D(0.0000, -4.6200, 0.0000), 1e-3)` (no ==). Else it is converting a bool to double and comparing it to 1e-3

Reply:
I fixed a few of these in #217 (https://github.com/chemfiles/chemfiles/pull/217/files#diff-ff5a736e4c9aa54737d30385699c44d2)

Reply:
Whoops! I wrote this during a plane flight, maybe I should be more careful with 'plane' code.